go/runtime/registry/registry.go

```diff
@@ -77,6 +79,13 @@ type Runtime interface {
 	// WatchRegistryDescriptor subscribes to registry descriptor updates.
 	WatchRegistryDescriptor() (<-chan *registry.Runtime, pubsub.ClosableSubscription, error)
 
+	// ActiveDescriptor waits for runtime to be initialized and then returns
```

Comment:
Maybe add a comment here (and below) saying that this may block until the next epoch transition.